server/routes/api.js

```diff
@@ -110,4 +109,46 @@ router.patch('/cards', (req, res) => {
     }
 })
 
+/*
+API call to get a single column from the neighborhood list using a constraint array.
+req: 1) the string name of the column to get data from.
+    2) a constraint_array of the same form as the api/getFilteredData endpoint.
+res: an array of numbers from the specified column with the specified constraints.
+*/
+router.get('/column', (req, res) => {
+    console.log("api/column call:")
+    console.log(req.body)
+
+    constraint_array = req.body.constraint_array;
```

Comment:
Gotta change this a bit due to the fact that React enforces a no-body GET request. We'll have to change this over to `.params`

Reply:
I'm reading up on the semantics of GET requests and came across these:
https://stackoverflow.com/a/983458
https://stackoverflow.com/a/10299780

Basically, there's nothing STOPPING a GET request from having a body, but it's generally understood that a GET request shouldn't do anything with the body of it's text.

Because we plan on using the body of our text for our function, it would be more proper to use a POST request, as we need the server to do something with that data

Reply:
I changed `api/column` to use POST instead of GET. POST is apparently more proper in the event that the server needs to use the body of the request. The consensus I've seen online is that GET requests should never use the body of a request.

Reply:
We should use GET. We are retrieving data, not modifying/creating.
Technically this should have also been done for retrieving the individual data points, but we (more) were pressed for time then.

https://www.w3schools.com/tags/ref_httpmethods.asp